cmd/config/docs/commands/merge3.md

```diff
@@ -0,0 +1,25 @@
+## merge3
+
+[Alpha] Merge Resource configuration files (3-way)
+
+### Synopsis
+
+[Alpha] Merge Resource configuration files (3-way)
+
+Merge (3-way) reads Kubernetes Resource yaml configuration files from source packages and updated 
+packages then writes the result to stdout and a destination package.
+
+Resources are merged using the Resource [apiVersion, kind, name, namespace] as the key.  If any of
+these are missing, merge will default the missing values to empty.
+
+Resources specified in the updated packages have higher-precedence and Resources specified
```

Comment:
If a field value differs between the ORIGINAL_DIR and UPDATED_DIR, the value from the UPDATED_DIR is taken and applied to the Resource in the DEST_DIR.